projects/ngx-openlayers/src/lib/sources/imagewms.component.ts

```diff
@@ -32,9 +36,9 @@ export class SourceImageWMSComponent extends SourceComponent implements OnChange
   @Input()
   serverType: string;
   @Input()
-  imageLoadFunction?: ImageLoadFunctionType;
+  imageLoadFunction?: LoadFunction;
   @Input()
-  logo: string | olx.LogoOptions;
+  logo: string; // TODO: (string | olx.LogoOptions);
```

Comment:
logo property does not exist anymore on ol 5.3.0

https://openlayers.org/en/latest/apidoc/module-ol_source_ImageWMS.html